### PR TITLE
[RA2] Ch02 -> Ch04 traceability for Elbrus branch

### PR DESCRIPTION
**Pull request by tomkivlin**

### CODE REVIEW

**Comment by tomkivlin** [comment]
Please don't delete the branch as also merging to master.

File: doc/ref_cert/RC2/chapters/chapter02.md [hunk 1]
```diff
@@ -172,14 +172,14 @@ are defined as mandatory (e.g. sec.std.001: The Cloud Operator **should**
...
 
-The following software versions are considered to verify Kubernetes v1.19
+The following software versions are considered to verify Kubernetes v1.20
 (latest stable release) selected by CNTT:
 
 | software                | version     |
 |-------------------------|-------------|
-| Functest                | kali        |
+| Functest                | leguer      |
 | kube-hunter             | 0.3.1       |
```
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter02.md:183]: should this be v0.4.1

File: doc/ref_cert/RC2/chapters/chapter02.md [hunk 2]
```diff
@@ -172,14 +172,14 @@ are defined as mandatory (e.g. sec.std.001: The Cloud Operator **should**
...
 | kube-bench              | 0.3.1       |
```
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter02.md:184]: should this be v0.5.0 -- added k8s 1.18 as default

File: doc/ref_cert/RC2/chapters/chapter02.md (hunk 1 above)
Comment (tomkivlin) [reply doc/ref_cert/RC2/chapters/chapter02.md:183]: Please ignore, this wasn't part of my change.

File: doc/ref_cert/RC2/chapters/chapter02.md (hunk 2 above)
Comment (tomkivlin) [reply doc/ref_cert/RC2/chapters/chapter02.md:184]: Please ignore, this wasn't part of my change.

**Comment by tomkivlin** [comment]
Created incorrectly, closing.

**Pull request closed without merging**